translate/canon/TranslateTextWithModel.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright(c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START translate_text_with_model]
+
+using Google.Cloud.Translation.V2;
+using System;
+
+public partial class TranslateSample
+{
+    public TranslationResult TranslateTextWithModel()
+    {
+        TranslationClient client = TranslationClient.Create();
+        TranslationResult result = client.TranslateText(
```

Comment:
Simon, how would you format this call?

Reply:
```csharp
client.TranslateText(
   text: "Hello World.",
   targetLanguage: "ja",
   sourceLanguage: "en",
   model: TranslationModel.NeuralMachineTranslation);
```

Reply:
I've incorporated your comments.  Now all examples use named parameters.  However, I still don't want to expressly prohibit comments on the parameters.  Sometimes the parameter name doesn't tell the whole story.  See the new code for an example.

Reply:
Thanks. I don't have strict objection on inline comments. I just prefer to use named parameters where applicable. Wherever the name of the parameter is not informative enough, we should probably alert the library creators so they improve it to make it easier to use without looking in documentation.